### PR TITLE
update ISSUE_TEMPLATE

### PR DESCRIPTION
**Pull request by BecodReyes**

### CODE REVIEW

**Comment by fatedier** [comment]
I have a question, how github decide which langauge template to use?

**Comment by BecodReyes** [comment]
> I have a question, how github decide which langauge template to use for different users?
> 
> And I'm prefer not to provide a Chinese version. It's not friendly for other peoples not familiar with Chinese. We'd better encourage our users communicate in English.

Thank you for your consideration for users who are not familiar with Chinese.

I haven't found a solution yet, so I created Chinese and English issue template. If you think this method is not good, I can delete the Chinese issue template.

**Comment by fatedier** [comment]
Yes, just provide a English version.

File: .github/ISSUE_TEMPLATE/config.yml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+blank_issues_enabled: false
+contact_links:
+  - name: DOCS
+    url: https://gofrp.org/en/docs/
```
Comment (fatedier) [inline .github/ISSUE_TEMPLATE/config.yml:4]: We haven't provide a English version here, just link repo README page?

Comment (BecodReyes) [reply .github/ISSUE_TEMPLATE/config.yml:4]: It was an oversight on my part, and I'm terribly sorry about it.

**Review by fatedier — APPROVED** [review]

**Pull request merged**